cmd/appgw-ingress/main.go

```diff
@@ -71,8 +76,15 @@ func main() {
 		ResourceGroup:  env.ResourceGroupName,
 		AppGwName:      env.AppGwName,
 	}
-	ctx := k8scontext.NewContext(getKubeClient(env), env.WatchNamespace, *resyncPeriod)
-	go controller.NewAppGwIngressController(appGwClient, appGwIdentifier, ctx).Start()
+	kubeClient := getKubeClient(env)
+	ctx := k8scontext.NewContext(kubeClient, env.WatchNamespace, *resyncPeriod)
+
+	recorder, err := getEventRecorder(kubeClient)
+	if err != nil {
+		glog.Fatalf("Error creating event recorder: %v", err.Error())
```

Comment:
This could actually be simplified to: `glog.Fatal("Error creating event recorder:", err)`